### ISSUE TITLE
 error: ‘class torch::nn::BatchNormImpl’ has no member named ‘running_var’

### ISSUE THREAD

**Issue by defenceVT** [Evangeb/yolov3#1]
hello, there was an error occured when i compile the project. the error is:
 error: ‘class torch::nn::BatchNormImpl’ has no member named ‘running_var’
batch_norm->running_var = batchNormVarTensor.view_as(batch_norm->running_var);

**Comment by UESTCtubiao** [comment]
Hello, thank you for sharing, what is the process of calling the CenterNet network training model in C++? What direction should I study? Looking forward to your reply.thanks.

**Comment by Evangeb** [comment]
This error is probably because the API changed and the variable names or methods changed.  I wrote it with an unstable version of Pytorch c++.
